specs/MultiProfile.md

```diff
@@ -0,0 +1,146 @@
+API spec for multiple profile support
+===
+
+# Background
+
+Currently, all WebView2s can only use one fixed Edge profile in the browser process, which is
+normally the **Default** profile or the profile specified by the **--profile-directory** command
+line switch. Regarding this we have got a bunch of requests to support multiple profiles, so we're
+adding these new APIs.
+
+With this you can have different WebView2s running with separate profiles under a given user data
+directory, which means separate cookies, user preference settings, and various data storage etc.,
+to enable you to build a more wonderful experience for your application.
```

Comment:
Previously the end developer could use more than one user data folder to have separate cookies and so on. With profile support you get that but also just one browser process? Is that the goal is reduced memory and CPU footprint from having to support multiple browser processes? Or are there other benefits or goals for this?

Reply:
Thanks! Added several sentences to make clear of the merit of our multiple profile support.